### ISSUE TITLE
Performance is slow on large textures

### ISSUE THREAD

**Issue by thgil** [thgil/Unity-Destructible-Sprite#1]
There is lots of places where the code can be sped up and optimized.

**Comment by skyboardsoftware** [comment]
I have worked on a version that uses come CoRoutines... Still much work is needed...

**Comment by thgil** [comment]
Yeah this code can be improved greatly.. Let me know how it goes!
